integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/mysql/CatalogMysqlIT.java

```diff
@@ -1072,4 +1072,58 @@ public void testBackQuoteTable() {
 
     Assertions.assertDoesNotThrow(() -> tableCatalog.dropTable(tableIdentifier));
   }
+
+  @Test
+  void testMySQLTableNameCaseSensitive() {
+    Column col1 = Column.of("col_1", Types.LongType.get(), "id", false, false, null);
```

Comment:
why mysql test NameCaseSensitive while pg test specialTableName? I think Mysql and PG need to test both cases.